src/material/datepicker/month-view.ts

```diff
@@ -252,6 +250,19 @@ export class MatMonthView<D> implements AfterContentInit, OnChanges, OnDestroy {
     this._changeDetectorRef.markForCheck();
   }
 
+  /** Handles when a new date becomes active. */
+  _dateBecomesActive(event: MatCalendarUserEvent<number>) {
```

Comment:
I agree that the `_dateBecomesActive` doesn't do much to describe what this function does.

I went with `_handleCalendarBodyDateFocused` to align with another function in this class that handles keydown events, `_handleCalendarBodyKeydown`. "Handle" seems appropriate because it does more than set a variable; it can also do side effects by firing the ActiveDateChange event and calling `_focusActiveCellAfterViewChecked` on the calendar body.